legacy/core/src/main/java/com/fsck/k9/K9.kt

```diff
@@ -383,7 +388,15 @@ object K9 : KoinComponent {
         isUseBackgroundAsUnreadIndicator = storage.getBoolean("useBackgroundAsUnreadIndicator", false)
         isShowComposeButtonOnMessageList = storage.getBoolean("showComposeButtonOnMessageList", true)
         isThreadedViewEnabled = storage.getBoolean("threadedView", true)
-        fontSizes.load(storage)
+
+        val disableFontSizeConfig = when (featureFlagProvider.provide("disable_font_size_config".toFeatureFlagKey())) {
+            FeatureFlagResult.Enabled -> true
+            FeatureFlagResult.Disabled -> false
+            FeatureFlagResult.Unavailable -> false
+        }
+        if (!disableFontSizeConfig) {
+            fontSizes.load(storage)
+        }
```

Comment:
This could be

```suggestion
        featureFlagProvider.provide("disable_font_size_config".toFeatureFlagKey())
            .onDisabledOrUnavailable {
                fontSizes.load(storage)
            }
```